py/server/deephaven/experimental/iceberg.py

```diff
@@ -270,48 +270,43 @@ def adapter(
             and `org.apache.iceberg.hadoop.HadoopFileIO`
 
     Other common properties include:
-    - `warehouse` - the location of the data warehouse.
+    - `warehouse` - the root path of the data warehouse.
     - `client.region` - the region of the AWS client.
     - `s3.access-key-id` - the S3 access key for reading files.
     - `s3.secret-access-key` - the S3 secret access key for reading files.
     - `s3.endpoint` - the S3 endpoint to connect to.
 
     Example usage #1 - REST catalog with an S3 backend (using MinIO):
     ```
-    from deephaven.experimental import s3, iceberg
+    from deephaven.experimental import iceberg
 
-    properties = {
+    adapter = iceberg.adapter_generic(name="generic-adapter", {
```

Comment:
```suggestion
    adapter = iceberg.adapter_generic(name="generic-adapter", properties={
```
In python it is generally considered bad form to follow a named parameter by an unnamed one.